task1/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+            Container(
+                color: Colors.orange,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
+                child: Center(
+                  child: Text("With",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+            Container(
+                color: Colors.tealAccent,
+                width: 250.0,
+                height: 100,
+                margin: EdgeInsets.all(30),
+                child: Center(
+                  child: Text("Flutter",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic, fontSize: 60.0)),
+                )),
+          ],
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        ),
+      ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          navigateToSubPage(context);
+        },
+        child: Icon(Icons.near_me),
+      ),
+    );
+  }
+}
+
+Future navigateToSubPage(context) async {
+  Navigator.push(context, MaterialPageRoute(builder: (context) => SubPage()));
+}
+
+class SubPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Back to Main Page'),
+        backgroundColor: Colors.blue,
+      ),
+      body: Column(
+        children: [
+          Row(
+            children: <Widget>[
+              Container(
+                color: Colors.indigoAccent,
+                width: 80.0,
+                height: 90,
+                margin: EdgeInsets.all(20),
+              ),
+              Container(
+                decoration: BoxDecoration(
+                    border: Border.all(
+                      color: Colors.black,
+                    ),
+                    gradient: LinearGradient(
+                        begin: Alignment.topRight,
+                        end: Alignment.bottomLeft,
+                        colors: [Colors.yellow, Colors.orangeAccent])),
+                width: 80.0,
+                height: 90,
+                margin: EdgeInsets.all(20),
+              ),
+              Container(
+                decoration: BoxDecoration(
+                    border: Border.all(
+                      color: Colors.black,
+                    ),
+                    gradient: LinearGradient(
+                        begin: Alignment.topRight,
+                        end: Alignment.bottomLeft,
+                        colors: [Colors.yellow, Colors.pink])),
+                width: 80.0,
+                height: 90,
+                margin: EdgeInsets.all(20),
+              ),
+            ],
+          ),
+          Image.asset(
```

Comment:
Once, you wrap the Image.asset with Expanded, remove the width and height parameter.